src/Pim/index.tsx

```diff
@@ -44,14 +44,14 @@ function objValues(obj: any) {
 }
 
 const itemsSelector = createSelector(
-  (props: {syncInfo: SyncInfo}) => props.syncInfo,
+  (props: { syncInfo: SyncInfo }) => props.syncInfo,
```

Comment:
Please don't mix style and logic changes in a PR, it makes things very hard to review. Same goes for almost all of the rest of this file.